src/Octoshift/Services/GithubApi.cs

```diff
@@ -158,6 +158,10 @@ public virtual async Task<bool> DoesRepoExist(string org, string repo)
         {
             return true;
         }
+        catch (HttpRequestException ex) when (ex.StatusCode == HttpStatusCode.MovedPermanently)
```

Comment:
An alternative approach would have been to update `GetNonSuccessAsync` (see lines above) to allow multiple "allowed HTTP statuses". But that turned out to be reasonably complex, and I'm not sure how likely it is that we'll need this again, so I took the easy way out.